juno/assetlist.json

```diff
@@ -2015,6 +2015,33 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/hera.png"
         }
       ]
+    },
+    {
+      "description": "Prop #16, Prop #20.",
+      "type_asset": "cw20",
+      "address": "juno1gwstg5ycwzxkecqat46kw3s0hwt6c9afsuvllu7cec0xh7nquhqszgf82c",
+      "denom_units": [
+        {
+          "denom": "cw20:juno1gwstg5ycwzxkecqat46kw3s0hwt6c9afsuvllu7cec0xh7nquhqszgf82c",
+          "exponent": 0
+        },
+        {
+          "denom": "takumi",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:juno1gwstg5ycwzxkecqat46kw3s0hwt6c9afsuvllu7cec0xh7nquhqszgf82c",
+      "name": "TAKUMI ASANO",
+      "display": "takumi asano",
```

Comment:
This value is invalid is not defined as a denom_unit. I think you mean to have just 'takumi'